src/diffusers/models/cross_attention.py

```diff
@@ -22,6 +24,7 @@
     LoRAXFormersAttnProcessor,
     SlicedAttnAddedKVProcessor,
     SlicedAttnProcessor,
+    TuneAVideoAttnProcessor,
```

Comment:
```suggestion
```

Reply:
Removed TuneAVideoAttnProcessor from attention.py and moved it into the pipeline file as it's specific to TuneAVideo